TESTING.md

```diff
@@ -145,44 +145,39 @@ uses the `RemoteLoggingHelper` to create a metric.
 
 ### Testing code that uses Pub/Sub
 
-#### On your machine
-
-You can test against a temporary local Pub/Sub by following these steps:
-
-1. Start the local Pub/Sub emulator before running your tests using `LocalPubSubHelper`'s `create`
-and `start` methods. This will bind a port for communication with the local Pub/Sub emulator.
-  ```java
-  LocalPubSubHelper helper = LocalPubSubHelper.create();
-
-  helper.start(); // Starts the local Pub/Sub emulator in a separate process
-  ```
+You can test against a Pub/Sub emulator:
 
-2. Create and use a `PubSub` object with the options given by the `LocalPubSubHelper` instance. For
-example:
-  ```java
-  PubSub localPubsub = helper.getOptions().getService();
-  ```
-
-3. Run your tests.
+1. [Install Cloud SDK](https://cloud.google.com/sdk/downloads)
 
-4. Stop the local Pub/Sub emulator by calling the `stop()` method, like so:
-  ```java
-  helper.stop();
-  ```
-
-#### On a remote machine
+2. Start the emulator:
+```shell
+$ gcloud beta emulators pubsub start
+```
 
-You can test against a remote Pub/Sub emulator as well. To do this, set the `PubSubOptions` project
-endpoint to the hostname of the remote machine, like the example below.
+To determine which host/port the emulator is running on:
+```shell
+$ gcloud beta emulators pubsub env-init
+# Sample output:
+#   export PUBSUB_EMULATOR_HOST=localhost:8759
```

Comment:
Should we also document that this is a valid shell command and so the user can run `$(gcloud beta emulators pubsub env-init)`? I'm leaning towards no, since the format should be familiar to Unix users. @garrettjonesgoogle ?